<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaEditar</class>
 <widget class="QMainWindow" name="TelaEditar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>350</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Editar</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>../Imagens/icone.png</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="Editar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QPushButton" name="PlanodeFundo">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-4</y>
      <width>650</width>
      <height>430</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <stylestrategy>NoAntialias</stylestrategy>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>../Imagens/grr segunda tela.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>850</width>
      <height>600</height>
     </size>
    </property>
    <property name="shortcut">
     <string notr="true"/>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="NOMEDOPRODUTO">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>28</y>
      <width>215</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 15pt &quot;COCOGOOSE &quot;;</string>
    </property>
    <property name="text">
     <string>NOME DO PRODUTO</string>
    </property>
   </widget>
   <widget class="QLabel" name="CODIGOLOTE">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>120</y>
      <width>147</width>
      <height>29</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 15pt &quot;COCOGOOSE &quot;;</string>
    </property>
    <property name="text">
     <string>CÃ“DIGO LOTE</string>
    </property>
   </widget>
   <widget class="QLabel" name="VALIDADE">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>118</y>
      <width>103</width>
      <height>29</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 15pt &quot;COCOGOOSE &quot;;</string>
    </property>
    <property name="text">
     <string>VALIDADE</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>414</x>
      <y>156</y>
      <width>151</width>
      <height>33</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Bahnschrift&quot;;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="correctionMode">
     <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
    </property>
    <property name="date">
     <date>
      <year>2019</year>
      <month>5</month>
      <day>3</day>
     </date>
    </property>
    <property name="maximumDate">
     <date>
      <year>2200</year>
      <month>12</month>
      <day>31</day>
     </date>
    </property>
    <property name="minimumDate">
     <date>
      <year>2019</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
    <property name="currentSection">
     <enum>QDateTimeEdit::DaySection</enum>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Salvar">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>258</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>../Imagens/btn_Salvar.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>200</width>
      <height>200</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelAlerta">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>214</y>
      <width>233</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 85, 255);
selection-color: rgb(255, 240, 250);
font: 16pt &quot;Bahnschrift&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Voltar">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>12</y>
      <width>53</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>../Imagens/btn_Voltar.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="PegarCodigo">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>156</y>
      <width>355</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255, 233);
font: 14pt &quot;Bahnschrift&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PegarNome">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>70</y>
      <width>541</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255, 233);
font: 14pt &quot;Bahnschrift&quot;;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
